packages/lesswrong/components/posts/ElicitBlock.tsx

```diff
@@ -187,16 +195,20 @@ const ElicitBlock = ({ classes, questionId = "IyWNjzc5P" }: {
     }
     ${getFragment("UsersMinimumInfo")}  
   `);
-  
-  const roughlyGroupedData = groupBy(data?.ElicitBlockData?.predictions || [], ({prediction}) => Math.floor(prediction / 10) * 10)
-  const finelyGroupedData = groupBy(data?.ElicitBlockData?.predictions || [], ({prediction}) => prediction)
+  const sortedPredictions = sortBy(data?.ElicitBlockData?.predictions || [], ({prediction}) => prediction)
+  const roughlyGroupedData = groupBy(sortedPredictions, ({prediction}) => Math.floor(prediction / 10) * 10)
+  const finelyGroupedData = groupBy(sortedPredictions, ({prediction}) => Math.floor(prediction))
   const maxSize = (maxBy(Object.values(roughlyGroupedData), arr => arr.length) || []).length
 
   return <div className={classes.root}>
     <div className={classes.histogramRoot}>
-      {times(10, (bucket) => <div key={bucket} className={classNames(classes.histogramBucket, {
-        [classes.histogramBucketCurrentUser]: roughlyGroupedData[`${bucket*10}`]?.some(({creator}) => currentUser && creator?.displayName === currentUser.displayName)
-      })}>
+      {times(10, (bucket) => <div key={bucket} 
+        className={classNames(classes.histogramBucket, {
+          [classes.histogramBucketCurrentUser]: roughlyGroupedData[`${bucket*10}`]?.some(({creator}) => currentUser && creator?.displayName === currentUser.displayName)
+        })}
+        onMouseEnter={() => roughlyGroupedData[`${bucket*10}`]?.length && setHideTitle(true)}
```

Comment:
If we're making this stateful/mouse-event driven, it'd be good to take the `UsersName` components out of the tree entirely, when they're not visible.

Reply:
I don't actually think so. I am just really tired of everything event-driven that is remotely dependent on mounting and unmounting nodes breaking all the time. When I experimented with removing these, with a lot of wiggling I could get it into a state where the `onMouseLeave` trigger didn't fire, and then everything is sad. 

I guess I switched my general philosophy to "just do as much as you possibly can in CSS, unless it gets really ridiculously expensive".

Reply:
For example, see: https://github.com/facebook/react/issues/4492

Reply:
Oh, interesting. That React bug may explain some things about stuck-hover problems we've been getting. And I see a way to fix `useHover` to thoroughly sidestep the issue.